component/sections/pricing/index.js

```diff
@@ -1,10 +1,13 @@
 import React from "react"
 import dynamic from "next/dynamic"
+import {Elements} from '@stripe/react-stripe-js';
+import {loadStripe} from '@stripe/stripe-js';
 
-function Pricing({ data }) {
-  const variant = data?.variants?.variant
-
+function Pricing({ data, published }) {
+  const variant = data?.variants?.variant  
+  const stripePromise = loadStripe("pk_test_51JDL3ZLisY7c4Rogu5n4AXHiExRPAnoBHWnKDGk9sC6rHz0dddWe4gEaoSxut0b8Hh1GY1c4sb9MMgaYTnQAoj9D00ebJ62hRD")
```

Comment:
Make this key configurable via env variable. Preferably use `NEXT_PUBLIC_STRIPE_PKEY`

Reply:
Still seeing this key as static!